packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryConfirmKey/index.tsx

```diff
@@ -181,8 +182,6 @@ const AccountRecoveryConfirmKey = ({
             setBannerMessage(localizedBannerMessage);
           }
         }
-      } finally {
-        setIsLoading(false);
```

Comment:
Why move this out of the `finally`, is it because if it's successful we expect to navigate?